### DISCUSSION TITLE
Inspector - Orange Revert to Default icon (RGB)

### DISCUSSION THREAD

**Discussion by shak2** [godotengine/godot-proposals#9878] [Editor]
I've been watching people streaming godot dev. The one thing they all waste time on is scanning through the inspector panels to find their edits.

There is "Expand non-default" but no one uses this because there is always the feeling that you will find your edit in just 1 more second.

PROPOSAL: I would like to be able to set the color of JUST the "revert to default" icon to an RGB color selection (orange for me).

Maybe an option in settings can be added for this?

**Comment by theraot** [comment]
Are you talking about this: 
![image](https://github.com/godotengine/godot-proposals/assets/1212509/73c45c43-2d1f-43de-9e4e-5392fe87c473)
Or about this: 
![image](https://github.com/godotengine/godot-proposals/assets/1212509/d708b9d5-a77d-475e-8a71-3b476c438a94)
?

The first one is the revert icon. But you mention "Expand non-default" as a solution... Which implies the group where the modified property they are looking for was not expanded, in which case they won't see the revert icon regardless of color because the group is collapsed... Intead they would be looking at the text saying how many changes they have... Which I'll grant you is less visible, In fact, I think it is less visible on purpose.

**Reply by shak2** [reply]
The first symbol, the revert icon. I want to make it orange.

**Comment by Calinou** [comment]
I had a PR that made modified properties more obvious in the inspector, but it was rejected: https://github.com/godotengine/godot/pull/41143

**Reply by Mickeon** [reply]
I'm not a fan still, but admittedly the revert icon _could_ use a different color to stand out...

**Comment by AThousandShips** [comment]
I'm not convinced that the icon itself would be more visible with a different colour, it doesn't blend in right now it's in its own distinct place

But regardless it should not be orange, that'd make people think it's an error or a problem, it being white communicates IMO that it's a neutral function and not something wrong or something that needs fixing

**Reply by shak2** [reply]
It doesn't have to be orange. Let users set the color in project settings. This can also work as a nice beginner tutorial to get users comfortable changing project settings.

**Reply by AThousandShips** [reply]
I don't think we should make tutorials like that, tutorials should be meaningful 

And I don't know of any other icons you can modify in that way, so it might need new features
